pkg/local_object_storage/metabase/select.go

```diff
@@ -128,7 +128,6 @@ func (db *DB) selectObjects(tx *bbolt.Tx, cid *cid.ID, fs object.SearchFilters)
 
 		addr, err := addressFromKey([]byte(a))
 		if err != nil {
-			// TODO: storage was broken, so we need to handle it
```

Comment:
think it is important

Reply:
It was fixed in #1118 on the shard level.